src/system/CollapsibleNotice/CollapsibleNotice.tsx

```diff
@@ -0,0 +1,176 @@
+/** @jsxImportSource theme-ui */
+
+/**
+ * External dependencies
+ */
+import * as Collapsible from '@radix-ui/react-collapsible';
+import classNames from 'classnames';
+import React, { useState, useId } from 'react';
+import { BiChevronDown } from 'react-icons/bi';
+import { MdWarning, MdCheckCircle, MdInfo, MdError } from 'react-icons/md';
+import { ThemeUIStyleObject } from 'theme-ui';
+
+/**
+ * Internal dependencies
+ */
+import { Flex, Heading, Card } from '../';
+
+interface CollapsibleNoticeIconProps {
+	color: string;
+	variant: ColorVariants;
+}
+
+export type NoticeProps = React.HTMLAttributes< HTMLDivElement > & {
+	children: React.ReactNode;
+	title: React.ReactNode;
+	sx?: ThemeUIStyleObject;
+	variant?: ColorVariants;
+	headingVariant?: React.ElementType;
+	className?: string;
+	defaultOpen?: boolean;
+	ariaContentId?: string;
+};
+type ColorVariants = 'warning' | 'error' | 'alert' | 'success' | 'info';
+
+const CollapsibleNoticeIcon = ( { color, variant }: CollapsibleNoticeIconProps ) => {
+	const sx = { color, flex: '0 0 auto' };
+	const size = 24;
+
+	switch ( variant ) {
+		case 'info':
+			return <MdInfo sx={ sx } size={ size } aria-hidden="true" />;
+		case 'error':
+			return <MdError sx={ sx } size={ size } aria-hidden="true" />;
+		case 'success':
+			return <MdCheckCircle sx={ sx } size={ size } aria-hidden="true" />;
+	}
+
+	// alert and warning will get the Warning icon
+	return <MdWarning sx={ sx } size={ size } aria-hidden="true" />;
+};
+
+export const CollapsibleNotice = React.forwardRef< HTMLDivElement, NoticeProps >(
```

Comment:
I would suggest using the same `<Notice />` component with a new prop to be collapsed or not. Creating a new component will be hard to maintain, and some parts of the code will be duplicated (like icons, variants, etc.)

Reply:
Is a different component though. We can re-factor the icons into a shared file but we have different formats..